src/routes/(site)/about/officer-profile.svelte

```diff
@@ -9,41 +17,168 @@
   export let info: Officer;
   export let placeholderPicture = 'placeholder.webp';
   export let dev = false;
```

Comment:
Note: Variable used to determine whether the dev server is enabled can be imported like so:
```ts
import { dev } from '$app/environment';
```
See: <https://kit.svelte.dev/docs/modules#$app-environment-dev>.

Reply:
> I'm actually not sure why this was here in the first place. I didn't touch it in any meaningful way.

All good. I believe it is being followed up in #612.